Source/DSPSpy/main_spy.cpp

```diff
@@ -69,58 +68,15 @@ u16 dspreg_in[32] = {
     0x0000, 0x0000, 0x0003, 0x0004, 0x8000, 0x000C, 0x0007, 0x0008, 0x0009, 0x000a,
 };  ///            ax_h_1   ax_h_1
```

Comment:
Does anyone know what this comment means?

Reply:
It was originally like this:

https://github.com/dolphin-emu/dolphin/blob/6ec5d28eda80083b17ca33516932be0bc4eb1eff/Source/DSPSpy/main_spy.cpp#L66-L71

where these sort of line up with registers AXH0 and AXH1... but the alignment isn't great and the comment uses `ax_h_1` twice instead of `ax_h_0` then `ax_h_1`. I don't think it's useful.

Reply:
I've removed this comment in the PR.